packages/techdocs-common/api-report.md

```diff
@@ -17,38 +17,42 @@ import { ScmIntegrationRegistry } from '@backstage/integration';
 import { UrlReader } from '@backstage/backend-common';
 import { Writable } from 'stream';
 
-// Warning: (ae-missing-release-tag) "DirectoryPreparer" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
+export type DirectoryFactory = {
```

Comment:
Maybe name it `DirectoryPreparerOptions`

Reply:
Renamed to PreparerConfig: https://github.com/backstage/backstage/pull/9718/commits/e257a8f0f499672b3e696df53d0604aba9ddbf0b;